src/passes/HashStringifyWalker.cpp

```diff
@@ -0,0 +1,56 @@
+#include "stringify-walker.h"
```

Comment:
This doesn't look like a pass, and AllCaps names in this directory are, I think, all passes atm. Can this perhaps be renamed to `stringify-walker.cpp` as it looks like it implements that header?

Reply:
you got it, done